goalgorilla/open_social
# Issue #3097276 by Kingdutch: Remove features from Social Group modules

<h2>Problem</h2>
See problem definition in [#3092272]. 

<h2>Solution</h2>
<ul>
  <li>Remove the <code>module.features.yml</code> file</li>
  <li>Move the default editable config from <code>hook_install()</code> into a module's <code>config/install/*.yml</code> file.
  <li>If <code>hook_install</code> alters configuration from other modules, consider whether we should include the config file in the profile's <code>config/install</code> folder to overwrite it instead</li>
  <li>Remove any _module_set_defaults function</li>
</ul>


## Issue tracker
https://www.drupal.org/project/social/issues/3097276

## How to test
<ul>
  <li>Enable all changed modules</li>
  <li>See that none of them install configuration provided by other modules</li>
  <li>Change configuration for each module</li>
  <li>Revert features</li>
</ul>
